app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -1,6 +1,7 @@
 package org.fossasia.susi.ai.activities;
 
 import android.Manifest;
+import android.app.Activity;
```

Comment:
@anantprsd5 please remove the unwanted import statement it's unused as by @codacy-bot